addons/point_of_sale/tests/test_frontend.py

```diff
@@ -479,9 +484,11 @@ def setUpClass(cls, chart_template_ref=None):
         env['ir.property']._set_default("property_product_pricelist", "res.partner", public_pricelist, main_company)
 
 
-@odoo.tests.tagged('post_install', '-at_install')
+@tagged('post_install', '-at_install')
 class TestUi(TestPointOfSaleHttpCommon):
     def test_01_pos_basic_order(self):
+        if not loaded_demo_data(self.env):
```

Comment:
I'm wondering if it wouldn't be better to let the test crash in nightly and ask the team to fix it. 🤔 
Otherwise we may forget it forever.
Maybe skip transversal tests like main flow but let module specific tests be red.

Reply:
After discussion: Log a warning instead. Purpose: Have a distinction between known and new issues.